cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/SavedAttachment.java

```diff
@@ -30,7 +30,7 @@
  *
  * @api_public
  */
-class SavedAttachment extends Attachment {
+public class SavedAttachment extends Attachment {
```

Comment:
I'm confused why this was API public before when the class itself wasn't public!